character-fx.js

```diff
@@ -246,6 +247,25 @@ class CharacterFx {
       }
     };
     _updateSonicBoomMesh();
+
+    const _updateHealEffectMesh = () => {
+      if (!this.healEffect ) {
+        this.healEffect = metaversefile.createApp();
+        (async () => {
+          await metaverseModules.waitForLoad();

```

Comment:
I think we can get rid of this. This code should not be running until the modules are loaded.